iot-firehose-s3-cdk/iot-kfh-s3/bin/iot-kfh-s3.ts

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import { IotKfhS3Stack } from '../lib/iot-kfh-s3-stack';
```

Comment:
Think you are missing the lib folder / files? Tried to run this locally but fails.